include/spdlog/logger.h

```diff
@@ -17,6 +17,7 @@
 #include <spdlog/common.h>
 #include <spdlog/details/log_msg.h>
 #include <spdlog/details/backtracer.h>
+#include <spdlog/details/fmt_helper.h>
```

Comment:
Lets try to avoid including this internal header . See https://github.com/gabime/spdlog/pull/2346#issuecomment-1106085031. 
Perhaps moving the “to_string_view()” functions to common.h

Reply:
Good idea, I'll find another place for them.